src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
-  As per the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components), kindly make sure to use functional components.